sdk/core/Azure.Core.TestFramework/src/Instrumentation/DiagnosticScopeValidatingInterceptor.cs

```diff
@@ -198,6 +199,8 @@ internal static async ValueTask<T> ValidateDiagnosticScope<T>(Func<ValueTask<(T
             using ClientDiagnosticListener diagnosticListener = new ClientDiagnosticListener(s => s.StartsWith("Azure."), asyncLocal: true);
             try
             {
+                // activities may be suppressed if they are called in scope of other activities create by other SDK methods. Let's unsuppress them.
```

Comment:
Would it be possible to add a comment saying why we need to unsuppress them in the test framework tests?  I expect others will have this question as well as me :)

Reply:
sure, done!